### ISSUE TITLE
Possible to get achievements with this?

### ISSUE THREAD

**Issue by tydog98** [auntiepickle/ff7seventhheavensteamdeck#4]
The Steam integration button doesn't seem to work  in Wine, any way around this?

**Comment by tydog98** [comment]
Actually I got it working using the built in Bottles Steam install script. For some reason it causes exceptions in 7th Heaven when DXVK is turned off, so you'll have to bumble around the menus with it on if you want this I guess (unless there's a better solution).

**Comment by dotaxis** [comment]
Did you manage to get it working in game mode?

I can't figure out how to launch Wine Steam and 7H at the same time, and the closest I've gotten totally broke controller input.

**Comment by tydog98** [comment]
I haven't used this in game mode. To get wine Steam and 7th Heaven running at the same time I added another shortcut to the Steam executable in the 7th Heaven bottle. Running the Steam exe with `-udpforce -allosarches -cef-force-32bit` launch parameters. You may need to use the `-noreactlogin` parameter to login the first time as wine doesn't seem to like the new login page.

**Comment by dotaxis** [comment]
Yeah I got it logged in and all but my controller input doesn't work 🤷‍♂️

**Comment by deterministicj** [comment]
For anyone else trying to get this working, I followed this guide to get Steam compatibility working with Proton directly in Steam gaming mode.

https://www.youtube.com/watch?v=Wfh-pE_TvGw

You still want to keep your other install as the mod catalog doesn't work using this at the moment. It's only used for launching the game with playtime and achievement working.

**Comment by dotaxis** [comment]
Thanks for the link. That's my guide 😅
